cmd/bigsky/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# Run this dockerfile from the top level of the indigo git repository like:
+#
+#   podman build -f ./cmd/bigsky/Dockerfile -t bigsky .
+
+### Compile stage
+
+# trouble with 'No such file or directory' when building using 'golang:1.19'
+# images, I think because of library errors when copied to 'alpine' image below
+# https://megamorf.gitlab.io/2019/09/08/alpine-go-builds-with-cgo-enabled/
+# TODO: lock to a specific version of go toolchain
+FROM alpine:3.17 AS build-env
+RUN apk add --no-cache build-base make go
+
+ADD . /dockerbuild
+WORKDIR /dockerbuild
+
+RUN go build -o /bigsky ./cmd/bigsky
+
+### Run stage
+FROM alpine:3.17
```

Comment:
We're building bigsky with cgo so are there any gotchas with running in alpine w/musl? I can't remember. I seem to recall wanting to do a few things like `export GODEBUG=netdns=go` when using CGO. But if things are working okay we may be able to look into this later.

Reply:
Maybe we need to install the tzdata package too? We should probably be building with `-tags timetzdata` https://pkg.go.dev/time/tzdata

Also, there's nothing keeping us on alpine I don't think. We can afford 100-200 MB container images if we run into annoyances. I do like Alpine though.

Reply:
These are good tips, i'll look in to these build flags and config vars. When deploying `beemo` to render I noticed the `netdns` thing but didn't dig in to it.

I'm a heavy debian user, and often like to have a regular linux distro so I have debug tools at hand in production if needed. In this case with a single binary it seemed nice and easy to keep image size small, and I mostly copied this from other templates that use alpine. And small size would be appealing for self-hosters. Can split between "our deployment dockerfile" and "public dockerfile" in the future if needed.